internal/termdbms/viewer/util.go

```diff
@@ -43,23 +44,25 @@ func TruncateIfApplicable(m *TuiModel, conv string) (s string) {
 }
 
 func GetStringRepresentationOfInterface(val interface{}) string {
-	if str, ok := val.(string); ok {
-		return str
-	} else if i, ok := val.(int64); ok { // these default to int64 so not sure how this would affect 32 bit systems TODO
-		return fmt.Sprintf("%d", i)
-	} else if i, ok := val.(int32); ok { // these default to int32 so not sure how this would affect 32 bit systems TODO
-		return fmt.Sprintf("%d", i)
-	} else if i, ok := val.(float64); ok {
-		return fmt.Sprintf("%.2f", i)
-	} else if i, ok := val.(float32); ok {
-		return fmt.Sprintf("%.2f", i)
-	} else if t, ok := val.(time.Time); ok {
+	switch t := val.(type) {
+	case string:
+		return t
+	case int64:
+		return fmt.Sprintf("%d", t)
+	case int32:
+		return fmt.Sprintf("%d", t)
+	case float32:
+		return fmt.Sprintf("%.2f", t)
+	case float64:
+		return fmt.Sprintf("%.2f", t)
+	case time.Time:
 		str := t.String()
 		return str
```

Comment:
Why not just `return t.String()`?

Reply:
Refactored here https://github.com/hazelcast/hazelcast-commandline-client/pull/130/commits/c11ffbd7e83a4dcdb4c01b2b2f9a92e26e7ad17d